packages/cli/main.js

```diff
@@ -2,23 +2,44 @@
 
 import hrp from "http-request-plus";
 import { format, parse } from "json-rpc-protocol";
+const WebSocket = require("ws");
+const util = require("util");
 
 async function main() {
   const params = require("minimist")(process.argv.slice(2));
   const method = params._[0];
 
-  const response = parse(
-    await hrp
-      .post("http://localhost:4000/api/", {
-        body: format.request(0, method, params),
-      })
-      .readAll("utf-8")
-  );
+  if (method === "displayEvents") {
+    let websocket = new WebSocket("ws://localhost:4000");
 
-  if (response.type === "response") {
-    console.log(response.result);
-  } else if (response.type === "error") {
-    console.error(response.error);
+    websocket.onerror = function(event) {
+      console.error(event);
+    };
+    websocket.onmessage = event => {
+      const message = parse(event.data);
+      if (message.type !== "notification") {
+        return;
+      }
+      console.log(
+        "method: %s\n %s\n",
+        message.method,
+        util.inspect(message.params, { showHiddent: true })
```

Comment:
Why not `JSON.stringify` ?

Reply:
You did not reply to this question, why did you decide to use `inspect` instead of `JSON.stringify`?
Any particular reason?

Reply:
There is no particular reason.
I change this, in the next commit.